npe2/manifest/schema.py

```diff
@@ -376,6 +378,44 @@ def _from_package_or_name(
 
     ValidationError = ValidationError  # for convenience of access
 
+    @classmethod
```

Comment:
put this here, but i'd be equally happy with putting it in `_from_npe1` and import PluginManifest there

Reply:
I prefer moving it to _from_npe1.py and keeping schema.py more specific to the npe2 schema.